src/napari_builtins/io/_read.py

```diff
@@ -27,6 +28,8 @@ def _alphanumeric_key(s: str) -> list[str | int]:
 
 URL_REGEX = re.compile(r'https?://|ftps?://|file://|file:\\')
 
+SCRIPTS_NAMESPACE = {}
```

Comment:
Shouldn't this be lower case since it is possibly modified later in the file on line 539? Alternatively change to `SETTING_SCRIPTS_NAMESPACE`

Reply:
Hm. I also was wrong and assumed that global variables should be uppercase, but after checking PEP-8 it is only for constants. 

On the other hand, uppercase variables are strictly signaling that we use a variable that is defined outside the function. And as we use the `[]` operator, the `global` is not used to signal it.